### PR TITLE
locked toggles (DEV-7315)

### PR DESCRIPTION
**Pull request by peter-horvath**

### CODE REVIEW

**Comment by luos** [comment]
:whale:

File: app/views/dashboard.html [hunk 1]
```diff
@@ -53,16 +53,16 @@ <h3 class="t-all">Features just for you</h3>
...
+                            {{ ctrl.getButtonContent(ctrl.isToggleActive( div.id )) }}
                         </button>
                     </div>
                 </div>
                 <div class="preview-toggle cf t-all" ng-repeat="div in user_features.withoutScreenshot | limitTo:8" ng-class-odd="'d2-d7'" ng-class-even="'d8-d13'"  >
                     <div class={{div.id}}>
                         <ng-include src="'views/feature.html'"></ng-include>
-                        <button type="button" ng-class="activated[div.name] ? 'active' : ''" class="button t6-t9 t-floatright" ng-class-odd="'d5-d7 d-padin'" ng-class-even="'d11-d13 d-padin'" ng-click="ctrl.setToggleActive(div.id, div.name, !activated[div.name], div.type)">
-                            {{ ctrl.getButtonContent(activated[div.name]) }}
+                        <button type="button" ng-disabled="ctrl.isToggleLocked( div.id )" ng-class="ctrl.isToggleActive( div.id ) ? 'active' : ''" class="button t6-t9 t-floatright" ng-class-odd="'d5-d7 d-padin'" ng-class-even="'d11-d13 d-padin'" ng-click="ctrl.setToggleActive(div.id, !ctrl.isToggleActive( div.id ), div.type)">
```
Comment (luos) [inline app/views/dashboard.html:64]: Good step, still could move ctrl.setToggleActive(div.id, !ctrl.isToggleActive( div.id ), div.type) into the ctrl as onActiveClick(div) or whatever but it's fine.

**Comment by luos** [comment]
Looks OK

File: app/views/dashboard.html (hunk 1 above)
Comment (peter-horvath) [inline app/views/dashboard.html:64]: Yeah I totally agree and you're right. However if you don't insist I won't do that refactoring now due to lack of time :S

**Pull request merged**